cfgov/jinja2/v1/_includes/molecules/related-posts.html

```diff
@@ -53,11 +62,13 @@ <h2 class="header-slug">
 
    Creates related posts markup when given:
 
+   settings:   An object used to customize the markup.
```

Comment:
Actually, since `view_more` is the only setting, why not just pass `view_more` directly?